### DISCUSSION TITLE
portal without password?

### DISCUSSION THREAD

**Discussion by stanelie** [khoih-prog/ESP_WiFiManager#106] [Q&A]
Hello.

I have not been able to find out how to configure the config portal without a wifi key. Is this possible?

Thanks.

**Comment by khoih-prog** [comment]
No. Without the PWD (key ???), how can the board connect to selected SSID ?

**Reply by stanelie** [reply]
I was not clear. I would like the config portal to generate an access point without security, without a key, so that anyone can connect to it and set it up.

Thanks.

**Reply by khoih-prog** [reply]
It's possible, but **strongly not advisable** because of security risk.

Imagine anybody can login into the Config Portal and change the settings with anything bad.

Anyway, to use **blank** PWD for Config Portal, for example in example [**ConfigOnDoubleReset**](https://github.com/khoih-prog/ESP_WiFiManager/tree/master/examples/ConfigOnDoubleReset), change this line from

https://github.com/khoih-prog/ESP_WiFiManager/blob/aeee669702f13058a47f34ddfd5a5937ff1df250/examples/ConfigOnDoubleReset/ConfigOnDoubleReset.ino#L815

into

```cpp
//password = "My" + ssid;
password = "";
```

Think hard before doing this **inadvisable** way and be prepared for the bad consequences.

Good Luck

**Comment by stanelie** [comment]
Hi.

For my use case in a low security requirement device, the benefits outweigh the downsides. This is what I was looking for, thanks!
